go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/google/pprof
```

Comment:
Did you mean to include these files? This doesn't seem to be related to this PR.

Reply:
It's better to add go module support. But I can delete this file if there is not go module support plan.